aichess.py

```diff
@@ -56,11 +59,14 @@ def __run_one_set_of_opponents(self, player1: ChessPlayer, player2: ChessPlayer,
             if winner is None:
                 draws += 1
 
-            # If winner is True, white won (player1)
-            elif winner:
+            # If winner is True and player1_starts is True, player1 won
+            # If winner is False and player1_starts is False, player1 won
+            elif winner == player1_starts:
                 player1_wins += 1
 
             total_moves += game.player_white.total_moves + game.player_black.total_moves
+
+            player1_starts != player1_starts
```

Comment:
should this be this way instead?
 player_1starts =  not player1_starts

Reply:
yes thank you